FreshdeskApi.Client/Tickets/Requests/CreateOutboundEmailRequest.cs

```diff
@@ -0,0 +1,106 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Linq;
+using FreshdeskApi.Client.CommonModels;
+using FreshdeskApi.Client.Tickets.Models;
+using Newtonsoft.Json;
+using TiberHealth.Serializer.Attributes;
+
+namespace FreshdeskApi.Client.Tickets.Requests
+{
+    [SuppressMessage("ReSharper", "UnusedMember.Global")]
+    public class CreateOutboundEmailRequest : IRequestWithAttachment
+    {
+        public CreateOutboundEmailRequest(TicketStatus status, TicketPriority priority, string? subject, string description,
+            string email, long emailConfigId, string? requesterName = null, string[]? ccEmails = null,
+            Dictionary<string, object>? customFields = null, DateTimeOffset? dueBy = null,
+            DateTimeOffset? firstResponseDueBy = null, long? groupId = null, string[]? tags = null,
+            long? companyId = null, string? ticketType = null,
+            IEnumerable<FileAttachment>? files = null)
+        {
+            Status = status;
+            Priority = priority;
+            RequesterName = requesterName;
+            Email = email;
+            Description = description;
+            CcEmails = ccEmails;
+            CustomFields = customFields;
+            DueBy = dueBy;
+            EmailConfigId = emailConfigId;
+            FirstResponseDueBy = firstResponseDueBy;
+            GroupId = groupId;
+            Tags = tags;
+            Subject = subject;
+            TicketType = ticketType;
+            Files = files;
+        }
+
+        /// Name of the requester
```

Comment:
Please use valid [documentation-comments](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments)

```suggestion
        /// <summary>
        /// Name of the requester
        /// </summary>
```

Reply:
The CreateOutboundEmailRequest was copied from CreateTicketRequest, which als doesn't have xml comments.
I have fixed them in CreateOutboundEmailRequest.